src/foam/u2/crunch/lab/CrunchLab.js

```diff
@@ -155,7 +155,8 @@ foam.CLASS({
           else
             this.currentMemento_ = null;
         }
-      }
+      },
+      menuKeys: ['admin.capabilities']
```

Comment:
by default this will be the daoSummary flow ... so not sure I understand why any of the properties are updated with menuKeys ... basically why are you setting 'admin.capabilities'?

Reply:
@Nauna Yes, the default is daoSummary flow, but you need to have service.{prop.daoKey} permission to enable the link to daoSummary view (you can take a look at the flow chart above). We can't give service.{prop.daoKey} to certain groups since admin will also get it and we might not want admin to have it. So instead I use menuKeys to control this. This is the left-most path in the flow chart.

Reply:
AHhh I see. Interesting!